viral/ensemble_reactivation.py

```
@@ -862,5 +1100,214 @@ def plot_speed_and_position_logic(trials: List[TrialInfo]) -> None:
+    )
+    shaded_line_plot(
+        response_re, x_axis=np.arange(-30, 30) / 30, color="blue", label="reactivation"
+    )
+    shaded_line_plot(
+        response_pre,
+        x_axis=np.arange(-30, 30) / 30,
+        color="orange",
+        label="preactivation",
+    )
+    plt.xlabel("Time (s) from event onset")
+    1 / 0
```

Ruff (0.13.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)